src/Command/DumpPackagesCommand.php

```diff
@@ -28,17 +29,20 @@ class DumpPackagesCommand extends Command
 {
     use \App\Util\DoctrineTrait;
 
-    private SymlinkDumper $dumper;
+    private SymlinkDumper $symlinkDumper;
+    private PackageDumper $packageDumper;
     private Locker $locker;
     private ManagerRegistry $doctrine;
     private string $cacheDir;
 
-    public function __construct(SymlinkDumper $dumper, Locker $locker, ManagerRegistry $doctrine, string $cacheDir)
+    public function __construct(SymlinkDumper $symlinkDumper, PackageDumper $packageDumper, Locker $locker, ManagerRegistry $doctrine, string $cacheDir)
     {
-        $this->dumper = $dumper;
-        $this->locker = $locker;
-        $this->doctrine = $doctrine;
-        $this->cacheDir = $cacheDir;
+        $this->symlinkDumper = $symlinkDumper;
+        $this->packageDumper = $packageDumper;
+        $this->locker        = $locker;
+        $this->doctrine      = $doctrine;
+        $this->cacheDir      = $cacheDir;
```

Comment:
there is no reason to align the `=` signs. such coding standard only increases the need for git conflicts due to reformatting. The Packagist source code mostly follows the Symfony coding standards, which have the explicit rule of *not* doing such alignment.

Reply:
Right, I didn't know about the coding standard, as nothing is enforced by phpcs or another tool.
I've reverted them!